packages/react-native-reanimated/src/layoutReanimation/web/transition/Jumping.web.ts

```diff
@@ -0,0 +1,51 @@
+'use strict';
+import type { TransitionData } from '../animationParser';
+
+const jumpHeight = 100;
```

Comment:
Here we can think about making `jumpHeight` dependent on `Views` height. We do not have access to it right now, but we can pass it in `transitionData` (or find better way to do this). Let me know what do you think 😅

Reply:
Let's check the native implementation

Reply:
Seems that native implementation uses `max` from distances between translation on X and Y axes 😅